### ISSUE TITLE
Sites Directory

### ISSUE THREAD

**Issue by mehulkaklotar** [buddyboss/buddyboss-platform#1475]
**Describe the bug**
When we enable multisite then we see Sites directory in pages and we have front end directory page for that which shows all sites and my sites tabs. Sometimes it shows sites directory pages settings and sometimes not. And there are other issues as well.

https://github.com/buddyboss/buddyboss-platform/issues/942
https://github.com/buddyboss/buddyboss-platform/issues/974

**To Reproduce**
1. On multisite setup, go to BuddyBoss > Pages.
2. It shows Sites Directory setting. For now we have dissabled this https://github.com/buddyboss/buddyboss-platform/pull/1472

**Expected behavior**
Need to decide whether to support this or not.

**Comment by searchboy-sudo** [comment]
Please support this. Its a crucial part of buddypress to allows users and groups to create their own sites.

**Comment by Dreamsync** [comment]
+1

Buddypress + Multisite enables you to create social blogging networks. That's incredibly powerful, and I think it's an idea who's time has finally come. Not only can users create their own sites, but you can also aggregate all of the network-wide activity on the main site. I see an opportunity here for Buddypress if they support and promote that. People are getting increasingly jaded on hyper-centralized social media, and niche blog networks/communities are on the rise.

**Comment by searchboy-sudo** [comment]
It seems the entire feature has been removes from the buddyboss feature entirely. It doesn't have a sites directory or even acknowledge that there are other sites in a global context...

**Comment by Dreamsync** [comment]
> It seems the entire feature has been removed from buddyboss entirely. It doesn't have a sites directory or even acknowledge that there are other sites in a global context...

That's a shame. Though I imagine they just don't yet see the benefit to putting energy into supporting networks.

**Comment by searchboy-sudo** [comment]
They know the value because they are some of the most experienced buddypress devs around. I thinks it’s more strategic than that.

**Comment by Dreamsync** [comment]
Right, my guess is that they see more money in single-site networks, as multisite tends to be pretty esoteric for most people. Hopefully they'll decide to start supporting it if enough people request it.

**Comment by sourabhmatolia** [comment]
@searchboy-sudo @Dreamsync 

We will work on supporting Multi-Site and this feature soon. 
Thanks for the feedback.

**Comment by Dreamsync** [comment]
> @searchboy-sudo @Dreamsync
> 
> We will work on supporting Multi-Site and this feature soon.
> Thanks for the feedback.

Awesome, that's music to my ears :)

Looking forward to seeing what you come up with..

**Comment by searchboy-sudo** [comment]
Any estimate when that would start? It's actually reallly dissapounted me to find out that it wasn't supported and has halted migration of my community to buddyboss platform and launch. Crossing my fingers.

**Comment by BoweFrankema** [comment]
Is there any news on this? It seems to have been completely removed in the latest versions of BuddyBoss, without any notice.. It was crucial for my project and I'd love to know what the status is?
